ui/packages/ui/src/app/shared/Utils.ts

```diff
@@ -349,7 +349,7 @@ export function minimizePropertyValues (propertyValues: Map<string, string>, pro
       if (propDefn.isMandatory) {
         minimizedValues.set(key, value);
       // Include non-mandatory if has default and value is different
-      }else if (propDefn.defaultValue) {
+      }else if (propDefn.defaultValue != null) {
```

Comment:
Hey @mdrillin can you maybe have a look into the backend code and fix sending any null to the UI too with that PR?

Reply:
Sure, but I'd suggest working that as a separate PR.  Is there a JIRA for it?

Reply:
You can just do it here in this PR. The purpose is the same.

Reply:
The purpose is not the same.  This PR eliminates sending of properties with default values from the UI to the backend.

Reply:
That's a question of the pov. "Fix issue with removal of default properties" can also mean: one side is fixing issues with such data in the frontend, but can also cover eliminating sending the wrong data in the first place from the backend.